src/txreconciliation.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_TXRECONCILIATION_H
+#define BITCOIN_TXRECONCILIATION_H
+
+#include <net.h>
+#include <sync.h>
+
+#include <tuple>
```

Comment:
8110da8b1b8bc83fa7fd2b76f9cc4688a7a02623

nit `#include <memory>` for `std::unique_ptr`.